.dev/docker/windows/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+
+# Use "--build-arg platform=x64" for 64 bit or x86 for 32 bit.
+ARG platform
+
+# Use to set specific commit to checkout
+ARG vcpkgcommit
+
+# Download channel for fixed install.
+ARG CHANNEL_BASE_URL=https://aka.ms/vs/16/release
+ADD ${CHANNEL_BASE_URL}/channel C:\TEMP\VisualStudio.chman
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Download and install Build Tools for Visual Studio 2019 for native desktop
+RUN wget $Env:CHANNEL_BASE_URL/vs_buildtools.exe -OutFile 'C:\TEMP\vs_buildtools.exe'; `
+    Start-Process -FilePath C:\TEMP\vs_buildtools.exe -ArgumentList `
+    "--quiet",                                                      `
+    "--norestart",                                                  `
+    "--nocache",                                                    `
+    "--installPath",                                                `
+    "C:\BuildTools",                                                `
+    "--wait",                                                       `
+    "--channelUri",                                                 `
+    "C:\TEMP\VisualStudio.chman",                                   `
+    "--installChannelUri",                                          `
+    "C:\TEMP\VisualStudio.chman",                                   `
+    "--add",                                                        `
+    "Microsoft.VisualStudio.Workload.VCTools",                      `
+    "--includeRecommended"                                          `
+    -Wait -PassThru;                                                `
+    del c:\temp\vs_buildtools.exe;                                  
+
+RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'));        `
+    choco install cmake git --installargs 'ADD_CMAKE_TO_PATH=System' -y --no-progress
+
+RUN git clone https://github.com/microsoft/vcpkg.git; cd vcpkg; git checkout $Env:commit;
```

Comment:
Is it supposed to be `commit` instead of VCPKGCOMMIT?

Reply:
No, its not 🤦